README.md

```diff
@@ -0,0 +1,109 @@
+# Test Definitions
+A test suite works with and without [LAVA](http://lavasoftware.org/). The following two
+sets of automated tests are supported.
+
+- `automated/linux/`
+- `automated/android/`
+
+For each test case, both test script and corresponding test definition files provided in
+the same folder that named after test case name. Test scripts are self-contained and
```

Comment:
For each test case, both the test script and the corresponding test definition files are provided in￼ the same folder and are named after the test case name.